packages/application-shell/src/components/login/login.js

```diff
@@ -170,70 +171,78 @@ export class Login extends React.PureComponent {
           return this.handleSubmit(values, addHandlers);
         }}
         render={formikProps => (
-          <LoginBox>
+          <Card
+            css={css`
+              width: 330px;
```

Comment:
Curious why we can't use the `m` constraint here from the design tokens? 🤔

(it's a question for the design team)

Reply:
I talked with Filip and this is really supposed to be a different value from the constraints.

Reply:
I'll talk to him...why the hell do we have the design tokens for?!